.gitignore

```diff
@@ -4,6 +4,7 @@ build/
 
 # Other files and folders
 .settings/
+.build.config.dist
```

Comment:
the `.dist` should be in the repo, and `build.config` should be ignored. then, the workflow is:

* clone repo 
* duplicate `.dist` file and remove `.dist` extension
* modify (gitignored) new file, this is what the build uses